spec/javascripts/views/story_view_spec.js

```diff
@@ -673,6 +673,14 @@ describe('StoryView', function() {
 
         expect(this.view.saveEdit).not.toHaveBeenCalled();
       });
+
+      it('update uploadInProgress variable', function() {
+        this.view.model.isNew = sinon.stub().returns(true);
+
+        this.view.attachmentDone();
+
+        expect(this.view.uploadInProgress).toBeFalsy();
```

Comment:
What is the initial value of this? If it's not set it'll also be falsy isn't it? You could set it to true, then call `attachmentDone` and assert is changed.